### PR TITLE
Removes gateway missions we don't and won't use

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do

This PR removes the 5 gateway maps that are broken, dysfunctional, and disabled.

## Why It's Good For The Game
These have been sitting in life-support on our codebase for a while, fixing them up has not happened despite ample opportunity to do so. Time we removed them.
If we want new gateway content something from scratch would be better than trying to fix these.

## Changelog
No player-facing changes.

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Please remove all the supporting content with these as well (stuff like the syndicate communicator remote for the wild west).

**Comment by S34NW** [comment]
> Please remove all the supporting content with these as well (stuff like the syndicate communicator remote for the wild west).

I don't actually know what is gateway mission-specific and not, but I can try to find what I can

File: code/modules/power/singularity/singularity.dm [hunk 1]
```diff
@@ -431,3 +431,16 @@
 	explosion(src.loc,(dist),(dist*2),(dist*4))
 	qdel(src)
 	return(gain)
+
+/obj/singularity/onetile
+	dissipate = 0
+	move_self = 0
+	grav_pull = 1
```
Comment (AffectedArc07) [inline code/modules/power/singularity/singularity.dm:438]:
```suggestion
/obj/singularity/onetile
	dissipate = FALSE
	move_self = FALSE
	grav_pull = TRUE
```

**Comment by S34NW** [comment]
and of course evil santa uses bloody removed stuff, thank god for CI

**Review by AffectedArc07 — APPROVED** [review]
If we ever want this back, we use git for a reason.

**Pull request merged**